sdk/identity/azure-identity/azure/identity/_credentials/on_behalf_of.py

```diff
@@ -98,7 +98,7 @@ def __init__(
                 raise ValueError('Specifying both "client_certificate" and "client_secret" is not valid.')
             try:
                 credential = get_client_credential(
-                    certificate_path=None, password=kwargs.pop("password", None), certificate_data=client_certificate
+                    certificate_path=None, password=kwargs.pop("password", None), certificate_data=client_certificate,send_certificate_chain=kwargs.pop("send_certificate_chain", False)
```

Comment:
These args should be split into separate lines due to line length, hence the CI linting failures.

To validate proper formatting, you can run the following from the `azure-identity` directory.

```
pip install tox
tox run -e black -c ../../../eng/tox/tox.ini --root . -- azure
```